src/GraphQL/Abstractions/IHasUserContext.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace GraphQL.Abstractions
+{
+    public interface IHasUserContext
```

Comment:
There is already `IProvideMetadata` interface so this one can be also named `IProvideUserContext`

Reply:
Hmm IHasX seems a bit more obvious, but if that's the convention im ok with that.